app/models/bo/client_id_lookup.rb

```diff
@@ -265,6 +266,69 @@ def call_with_retry settings, message_options
       return response
     end
 
+    def fetch_disability_verifications one_off: false
+      return unless @config.disability_verification_cuid.present?
+      msg = "Fetching disability verifications"
+      Rails.logger.info msg
+      @notifier.ping msg if send_notifications && msg.present?
+      settings = {
+        url: "#{@config.url}?wsdl=1&cuid=#{@config.disability_verification_cuid}",
+        method: :disability_lookup,
+      }
+      message_options = {
+        touch_point_id: @config.disability_touch_point_id,
+        touch_point_question_id: @config.disability_touch_point_question_id,
+      }
+      if one_off
+        soap = Bo::Soap.new(username: @config.user, password: @config.pass)
+        response = soap.disability_lookup(
+          settings[:url],
+          touch_point_id: @config.disability_touch_point_id,
+          touch_point_question_id: @config.disability_touch_point_question_id
+        )
+      else
+        response = call_with_retry(settings, message_options)
+      end
+    end
+
+    def set_disability_verifications
+      return unless @config.disability_verification_cuid.present?
+      @disability_verifications = fetch_disability_verifications.
+        group_by{ |row| row[:participant_enterprise_identifier].gsub('-', '') }
+      personal_ids = @disability_verifications.keys
+      source_clients = GrdaWarehouse::Hud::Client.source.where(
+        data_source_id: @data_source_id,
+        PersonalID: personal_ids
+        ).select(:id, :PersonalID, :disability_verified_on)
+      updated_source = 0
+      updated_destination = 0
```

Comment:
I would prefer these names to make it clear they are counts

Reply:
done